ladybug_comfort/cli/_helper.py

```diff
@@ -170,7 +171,10 @@ def thermal_map_csv(folder, temperature, condition, condition_intensity):
         'condition': os.path.join(folder, 'condition.csv'),
         'condition_intensity': os.path.join(folder, 'condition_intensity.csv')
     }
-    _data_to_csv(temperature, result_file_dict['temperature'])
-    _data_to_csv(condition, result_file_dict['condition'])
-    _data_to_csv(condition_intensity, result_file_dict['condition_intensity'])
+    with open(result_file_dict['temperature'], 'wb') as fp:
+        np.save(fp, temperature)
+    with open(result_file_dict['condition'], 'wb') as fp:
+        np.save(fp, np.array(condition, dtype=np.int8))
+    with open(result_file_dict['condition_intensity'], 'wb') as fp:
+        np.save(fp, np.array(condition_intensity, dtype=np.int8))
```

Comment:
This one should not be an int. I know it's an int for the UTCI recipe but all of the other thermal mapping recipes use floats for this one.